### ISSUE TITLE
Refactor public IP Pool with label

### ISSUE THREAD

**Issue by girikuncoro** [mahakamcloud/mahakam#46]
After #45 , we will have multiple ip pools tagged with specific usecases. Relevant functions must obtain ip pools from the labels, i.e. `ippool/group: publicip` and `ippoo/group: storage`

**Comment by girikuncoro** [comment]
This is no longer relevant given we will throw away ISCSi, and there will no longer be a Storage Network. We are going away from remote storage solution as per ajey's feedback.

Closing this
